cmake/FindISPC.cmake

```diff
@@ -32,7 +32,7 @@ set(ISPC_VERSION 1.21.0)
             COMMAND ${7ZA} x -aoa -o* ${CMAKE_SOURCE_DIR}/External/ispc/${ISPC_DIR}.zip
         )
     endif()
-    cmake_print_variables(CMAKE_SYSTEM_PROCESSOR)
+    cmake_print_variables(CMAKE_GENERATOR_PLATFORM)
```

Comment:
Same question as above, do you want to keep this log?

Reply:
Let's keep it for now. I used it to confirm the build environment, whether AMD64 or ARM64. It is also no harm to have some log info in build process.